autograder/submitty_autograding_shipper.py

```diff
@@ -37,6 +37,58 @@
 JOB_ID = '~SHIP~'
 
 
+# This function is copied, would rather use the version in this file instead.
+# import sys
+# sys.path.append('/usr/local/submitty/sbin/shipper_utils/')
+# import ssh_jump_proxy
+def ssh_connection_allowing_jump_proxy(target_user, target_host):
+
+    # load and parse the ssh config
+    my_key_filename = None
+    intermediate_host = None
+    intermediate_user = None
+    config_file = os.path.expanduser("~")+"/.ssh/config"
+    if (os.path.exists(config_file)):
+        conf = paramiko.SSHConfig()
+        conf.parse(open(config_file))
```

Comment:
A with statement would be preferable here to ensure the file is closed.

Reply:
changed!